packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -149,50 +153,56 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
       disabled = isDisabled;
     }
 
+    const ariaActiveDescendant = disabled
+      ? undefined
+      : isOpen && selectedIndices.length === 1 && selectedIndices[0] >= 0
+      ? this._listId + selectedIndices[0]
+      : undefined;
+
     return (
       <div className={ css('ms-Dropdown-container') }>
         { label && (
-          <Label className={ css('ms-Dropdown-label') } id={ id + '-label' } htmlFor={ id } ref={ this._dropdownLabel } required={ required }>{ label }</Label>
+          <Label className={ css('ms-Dropdown-label') } id={ this._labelId } htmlFor={ id } ref={ this._dropdownLabel } required={ required }>{ label }</Label>
         ) }
         <div
           data-is-focusable={ !disabled }
           ref={ this._dropDown }
           id={ id }
           tabIndex={ disabled ? -1 : 0 }
-          aria-expanded={ isOpen ? 'true' : 'false' }
           role='combobox'
-          aria-autocomplete='none'
-          aria-live={ disabled || isOpen ? 'off' : 'assertive' }
-          aria-label={ ariaLabel }
-          aria-describedby={ id + '-option' }
-          aria-activedescendant={ isOpen && selectedIndices.length === 1 && selectedIndices[0] >= 0 ? (this._id + '-list' + selectedIndices[0]) : null }
+          aria-haspopup = 'listbox'
+          aria-expanded={ isOpen ? 'true' : 'false' }
+          aria-describedby={ this._optionId }
+          aria-labelledby={ this._labelId }
+          aria-activedescendant={ ariaActiveDescendant }
           aria-disabled={ disabled }
-          aria-owns={ isOpen ? id + '-list' : null }
+          aria-owns={ isOpen ? this._listId : null }
           { ...divProps }
           className={ css(
             'ms-Dropdown',
             styles.root,
             className,
-            isOpen! && 'is-open',
+            isOpen && 'is-open',
             disabled! && ('is-disabled ' + styles.rootIsDisabled),
             required! && 'is-required',
           ) }
           onBlur={ this._onDropdownBlur }
           onKeyDown={ this._onDropdownKeyDown }
           onKeyUp={ this._onDropdownKeyUp }
           onClick={ this._onDropdownClick }
+          onFocus={ this._onFocus }
         >
           <span
-            id={ id + '-option' }
             className={ css(
               'ms-Dropdown-title', styles.title,
               !selectedOptions.length && 'ms-Dropdown-titleIsPlaceHolder',
               !selectedOptions.length && styles.titleIsPlaceHolder,
               (errorMessage && errorMessage.length > 0 ? styles.titleIsError : null))
             }
             aria-atomic={ true }
-            role='listbox'
-            aria-readonly='true'
+            aria-live='off'
```

Comment:
The `aria-live` fix you had here earlier actually looked quite good to me (unless you removed it for some other comment that I missed) 😅. While often aria-live is misused, in this case the logic you used for it locked it down well, and I think it should give some benefit for screen reader users changing the value with the combo collapsed

Reply:
Will keep the previous design.